README.md

```diff
@@ -343,9 +438,9 @@ The currently defined Metrics are:
 
 Labels are currently not set but are added for future use, e.g setting the error code for fetch errors etc.
 
-## Creating Custom Handlers
+## Extending Oryx
 
-It's easy to extend Oryx with your own HTTP handlers.
+It's easy to extend Oryx with your own context builders and HTTP handlers. Everything is functions so you
```

Comment:
"Everything is functions so you"
Missing something here?

Reply:
Fixed